src/customForms/components/UrlField.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <span class="form-control shell">
+    <span class="field control" :class="model.cssClass">
+      <input
+        :id="`form-${model.key}`"
+        v-model="field"
+        :name="model.key"
+        type="text"
+        :size="model.size"
+        @input="updateText()"
+      />
+    </span>
+    <span
+      style="opacity: 0.7;font-size: 1em;font-style: italic;padding: 5px;display: inline-block;"
+      class="icon-alert-triangle hint"
+      v-if="warn"
+    >
+      This url may be broken, check again or proceed with caution.
```

Comment:
I think we should tell the user when a url 200s. So three states, not checked (show nothing), checked and valid (green check), checked and not valid (yellow warning). Can we just make this a hover text and have the icon to the right?

My suggestion for the comment would be something like, "url could not be reached", or "no response from url". broken seems alarming.

Reply:
WORD